llvm-to-smt/llvm-passes/LLVMToSMT/FunctionEncoder.cpp

```diff
@@ -1988,6 +2006,8 @@ void FunctionEncoder::handleMemoryPhiNode(MemoryPhi &mphi, int passID) {
              << phiResolveI.to_string().c_str() << "\n";
       BBAsstVecIter->second.push_back(phiResolveI);
     }
+  } else {
+    throw std::runtime_error("[handleMemoryPhiNode] Unknown passID\n");
```

Comment:
If we used a `switch` statement here instead of `if/else`, I believe the compiler would be able to tell us if any `enum` entry is missing and we wouldn't need this fallback case anymore.

Reply:
That's a much better idea, I agree.